cgi/mapcache.c

```diff
@@ -126,7 +126,9 @@ static void fcgi_write_response(mapcache_context_fcgi *ctx, mapcache_http_respon
     int i;
     for(i=0; i<elts->nelts; i++) {
       apr_table_entry_t entry = APR_ARRAY_IDX(elts,i,apr_table_entry_t);
-      printf("%s: %s\r\n", entry.key, entry.val);
+      if (!strcasecmp(entry.key, "Content-Type")) {
```

Comment:
Don't you mean `Content-Length` ?

Reply:
I did mean to put `Content-Length`, however when setting that it crashes the process..!

![image](https://user-images.githubusercontent.com/490840/132216293-0fa25375-f761-494b-a15d-e1a897ceb4d0.png)

In fact removing the entire `printf("%s: %s\r\n", entry.key, entry.val);` for all headers here also fixes the issue, whilst other requests seem to work without issue. I'll investigate why it works removing the addition of all headers. 

I based the approach on the nginx update at https://github.com/MapServer/mapcache/blob/7bc0ad9a858b85ef8ee2b16c2fc82c1bf43935df/nginx/ngx_http_mapcache_module.c#L104